canarytokens/aws_infra/operations.py

```diff
@@ -137,37 +148,40 @@ def _build_operation_payload(
     return payload
 
 
-# TODO: add handle exist for token validation
 async def get_handle_response(handle_id: str, operation: AWSInfraOperationType):
     """
     Check if a response has been added to the specified handle in the redis DB and return it.
     """
     handle = queries.get_aws_management_lambda_handle(handle_id)
+    default_error = AWSInfraServiceError.REQ_HANDLE_INVALID
+    default_error_message = service_error_map.get(default_error)
```

Comment:
I'd rather this breaks loudly and close to home if we accidentally remove that error from the map, rather than becoming `None` and breaking somewhere else.
```suggestion
    default_error_message = SERVICE_ERROR_MESSAGE_MAP[default_error]
```